firetweet/src/main/java/org/getlantern/firetweet/model/Lantern.java

```diff
@@ -75,4 +91,27 @@ public static void stop() {
             }
         }
     }
+
+    public static void checkNewVersion() {
+        if (context != null) {
+            try {
+                PackageInfo pInfo = context.getPackageManager().getPackageInfo(context.getPackageName(), 0);
+                String version = pInfo.versionName;
+                String latestFireTweetVersion = Flashlight.GetFireTweetVersion();
+
+                Log.d(LOG_TAG, "Latest version of FireTweet is " + Flashlight.GetFireTweetVersion() +
+                        "; currently running version of FireTweet: " + version);
+
+                if (latestFireTweetVersion != null && !latestFireTweetVersion.equals(version)) {
```

Comment:
It's possible that the version string in cloud config is not updated when we test new version of FT, and logic here will trigger an update prompt. Will it cause some trouble?